gocontext.go

```diff
@@ -130,6 +143,11 @@ func CaptureError(ctx context.Context, err error) *Error {
 		}
 		e := tx.tracer.NewError(err)
 		e.Handled = true
+		bc := bodyCapturerFromContext(ctx)
+		if bc != nil {
+			e.Context.SetHTTPRequest(bc.request)
```

Comment:
this will be the pre-copy version of the request, so PostForm won't be set on this one...
I think this fine for now